docs/references.rst

```diff
@@ -1,5 +1,65 @@
 References
 ==========
 
+Bibliography
+------------
 .. bibliography::
     :cited:
+
+Glossary
```

Comment:
Hi @michalk8 do you want me to fill this in this PR or in a follow-up PR?

Reply:
Would be great to add it here!